api/routes.py

```diff
@@ -198,7 +198,7 @@ def initalize_metadata_model(schema_url):
 def get_temp_jsonld(schema_url):
     # retrieve a JSON-LD via URL and store it in a temporary location
     with urllib.request.urlopen(schema_url) as response:
-        with tempfile.NamedTemporaryFile(delete=False, suffix=".jsonld") as tmp_file:
+        with tempfile.NamedTemporaryFile(delete=False, suffix=".model.jsonld") as tmp_file:
```

Comment:
@linglp I wanted to check on this change with you. I couldn't use the existing `/visualize/attributes` endpoint and found there were issues with the file names. Later in the code, it was trying to split the string based on `.model.jsonld` so I made this change to reflect what was being looked for. I was able to use the endpoint afterwards but wanted to review with you before merge

Reply:
@GiaJordan Could you double check by running `tests/test_api.py`? If all the tests passed, then it should be okay.

Reply:
@linglp I've never been able to successfully run the api tests on my machine, even before the change. But it looks like the same tests are failing as normal

Reply:
You are not able to run any tests in test_api.py or only specific tests?

Reply:
I am able to run all of them, but some consistently fail and have since they were introduced

Reply:
Continued in [comment below](https://github.com/Sage-Bionetworks/schematic/pull/1195#issuecomment-1512174274)